frontend/src/scenes/exports/CreateExport.tsx

```diff
@@ -154,6 +167,9 @@ export function CreateS3Export(): JSX.Element {
 
             <PureField label="Frequency">
                 <LemonSelect
+                    onSelect={(value: 'hour' | 'day') => {
```

Comment:
nit: I prefer the adjective form ("hourly" and "daily" instead of "hour" and "day"). But with the changes to the models, changing this is probably not that trivial, so if considered better we can do a more comprehensive change later.

Reply:
It is a bit confusing here equating frequency and interval, I'm not too keen on that